File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/cior/d/ReactExtOrsAndStopOrsDO.java
package iih.ci.ord.cior.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 排斥扩展医嘱与排斥停止医嘱 DTO数据 
 * 
 */
public class ReactExtOrsAndStopOrsDO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 排斥扩展医嘱聚集数据
	 * @return FArrayList
	 */
	public FArrayList getReactextdos() {
		return ((FArrayList) getAttrVal("Reactextdos"));
	}
	/**
	 * 排斥扩展医嘱聚集数据
	 * @param Reactextdos
	 */
	public void setReactextdos(FArrayList Reactextdos) {
		setAttrVal("Reactextdos", Reactextdos);
	}
	/**
	 * 关联要停止的医嘱
	 * @return FArrayList
	 */
	public FArrayList getStopordos() {
		return ((FArrayList) getAttrVal("Stopordos"));
	}
	/**
	 * 关联要停止的医嘱
	 * @param Stopordos
	 */
	public void setStopordos(FArrayList Stopordos) {
		setAttrVal("Stopordos", Stopordos);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/cior/d/ReactExtOrsAndStopOrsDO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.cior.d
{
    /// <summary>
    /// ReactExtOrsAndStopOrsDO 的摘要说明。
    /// </summary>
    public class ReactExtOrsAndStopOrsDO : BaseDTO {

        public ReactExtOrsAndStopOrsDO() {
 
        }

        /// <summary>
        /// 排斥扩展医嘱聚集数据
        /// </summary>
		public FArrayList Reactextdos {
            get { return getAttrVal<FArrayList>("Reactextdos",null); }
            set { setAttrVal<FArrayList>("Reactextdos", value); }
        }

        /// <summary>
        /// 关联要停止的医嘱
        /// </summary>
		public FArrayList Stopordos {
            get { return getAttrVal<FArrayList>("Stopordos",null); }
            set { setAttrVal<FArrayList>("Stopordos", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Reactextdos", "Stopordos"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.cior.d.ReactExtOrsAndStopOrsDO";
        }
    }
}
